ext/debug/ctf/Kconfig

```diff
@@ -0,0 +1,18 @@
+#
+# Copyright (c) 2018 Oticon A/S
+#
+# SPDX-License-Identifier: Apache-2.0
+#
+
+config CTF
+	bool
+	prompt "Tracing via Common Trace Format support"
+	default n
+	select THREAD_MONITOR
+	select TRACING
+
+config CTF_BOTTOM_POSIX
+	bool
+	prompt "CTF backed by POSIX fwrite"
```

Comment:
I would maybe better describe it as something like "backend for the native_posix port, using a file in the host filesystem" or so. (POSIX APIs to Zephyr are being added to Zephyr, so it may otherwise be a bit too ambigious)